package.json

```diff
@@ -14,6 +14,8 @@
   "author": "OpenZipkin <openzipkin.alt@gmail.com>",
   "repository": "https://github.com/openzipkin/zipkin-js",
   "devDependencies": {
+    "babel-cli": "^6.18.0",
+    "babel-preset-es2015": "^6.18.0",
```

Comment:
ok, makes sense. 
but pr was opened for browser specific packages support and running transpiling step before publishing to npm